### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by Sevenpros**
Add responsiveness for desktop devices
justify contents to fit well on the bigger screen
adjust font size and other element to fit screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,138 +55,167 @@ <h2>I'am a Software Developer</h2>
...
-                        <li><a href="#">Ruby</a></li>
-                    </ul>
-                    <button class="project-button mobile-button">See Project</button>
-                </div>
-            </article>
+                </article>
+            </div>
         </section>
         <section class="about-section">
             <header class="about-header">
```
Comment (SajjadAhmad14) [inline index.html:200]: Please add a proper margin between `About me` and the horizontal line, and also between the `Languages` section and the horizontal line. Moreover, there should be only one horizontal line between the `About me` and `Languages` sections.

Yours:
hori-you

Design:
hori-o

File: index.html [hunk 2]
```diff
@@ -214,6 +254,13 @@ <h3>About Me</h3>
                     <input type="submit" class="project-button mobile-button contact-btn" value="Get in touch">
                 </form>
             </article>
+            <div class="social-icons footer-icons">
```
Comment (SajjadAhmad14) [inline index.html:256]: Please use the same color for icons for the desktop version as defined in the design. The color is `#505F79`.

Yours:
icons-you

File: style.css [hunk 3]
```diff
@@ -400,3 +429,129 @@ textarea::placeholder {
...
+@media (min-width: 768px) {
+  .menu-bar-desktop {
+    display: flex;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .headline {
```
Comment (SajjadAhmad14) [inline style.css:464]: The background image of the headline is overlapping the text on the small mobile view. Kindly make sure that the mobile version works the same as expected and according to the design.

back

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Seth, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by vic778 — APPROVED** [review]
Hi @Sevenpros   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**